packages/console/src/components/UserInfoCard/index.tsx

```diff
@@ -7,23 +7,24 @@ import * as styles from './index.module.scss';
 
 type Props = {
   className?: string;
-  user: Partial<
+  user?: Partial<
```

Comment:
just double check, does this pr have a screenshot of the testing result? ok if no for this one though - it'll be great to have one for ui-related changes

Reply:
Yes, this was funny. I actually prepared a screenshot when I was drafting the PR, but then I accidentally copied something else and lost the screenshot in pasteboard. Then I was like, "F it. I'm going to create the PR anyway..."
So...

lol, I'm updating screenshots right now

Reply:
lol thank you old tea